src/main/java/seedu/lifeasier/parser/Parser.java

```diff
@@ -185,9 +307,248 @@ private Command parseDisplayScheduleCommand(String input) {
         return new DisplayScheduleCommand(toDisplay);
     }
 
+    /**
+     * Parses the user Y/N inputs.
+     *
+     * @param input String containing the user's input.
+     * @param ui Input and output interaction with the user.
+     * @return A "Y" or "N" string.
+     */
+    public String parseUserInputYesOrNo(String input, Ui ui) {
+        LOGGER.log(Level.INFO, "Start check for Y/N input");
+        while (!input.trim().equals("Y") && !input.trim().equals("N")) {
```

Comment:
Can simplify the while loop conditionals
```suggestion
        while (!isEqualY && !isEqualN)
```